app/Notifications/RequestPaymentForCreationNotification.php

```
@@ -4,18 +4,17 @@
-use App\Models\CashRequest;
+use App\Models\PaymentRequest;
 use Illuminate\Bus\Queueable;
 use Notification;
 
-class RequestCashVoucherForDeniedNotification extends Notification
+class RequestPaymentForCreationNotification extends Notification
 {
     use Queueable;
     private $token;
     private $model;
     public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)